release/pipline/dags/istio_common_dag.py

```diff
@@ -161,7 +168,7 @@ def GenerateTestArgs(**kwargs):
     config_settings['GCS_FULL_STAGING_PATH'] = '{}/{}'.format(
         config_settings['GCS_STAGING_BUCKET'], gcs_path)
     config_settings['ISTIO_REPO'] = 'https://github.com/{}/{}.git'.format(
-        config_settings['GITHUB_ORG'],  config_settings['GITHUB_REPO'])
```

Comment:
extra space intentional?

Reply:
we removed extra space, what is remaining is intentional